Gifski/ConversionCompletedViewController.swift

```diff
@@ -16,6 +16,13 @@ final class ConversionCompletedViewController: NSViewController {
 	private var conversion: Gifski.Conversion!
 	private var gifUrl: URL!
 
+	private let tooltip = Tooltip(
+		identifier: "conversionCompletedTips",
+		text: "You can drag the below thumbnail as if it's a file, for example, to Finder or Safari. You can press Space to enlarge the preview.",
```

Comment:
just throwing some suggestions, feel free to ignore 😄 
```suggestion
		text: "Go ahead and drag the thumbnail to the Finder or Safari! You can also press Space to enlarge the preview.",
```